blockstack_client/rpc.py

```diff
@@ -3300,6 +3337,20 @@ def _dispatch(self, method_name):
                     },
                 },
             },
+            r'^/v1/zonefile$': {
+                'routes': {
+                    'POST' : self.POST_raw_zonefile
+                },
+                'whitelist': {
+                    'POST': {
```

Comment:
These security settings will mean that anyone in the world can use this endpoint to broadcast a zone file.  Just checking that this the intended behavior?  I can see it for e.g. core.blockstack.org, for example.

Reply:
Yep, the intention is for this to be a public endpoint.